geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und/oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet, auf welche Weise Stimmen gezählt oder geschätzt und ob sie anschließend protokolliert werden.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## § 5 Anträge
+
+(1) Anträge sind schriftlich zu verfassen und innerhalb von 14 Tagen nach Zugang der Einladung zur Vollversammlung bei den einladenden Personen einzureichen.
+
+(2) Die einladenden Personen entscheiden über die Aufnahme des Antrags in die Tagesordnung.
+
+(3) Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge werden vor der Abstimmung des jeweiligen Antrags behandelt.
+
+(2) Änderungsanträge können von der den ursprünglichen Antrag stellenden Person direkt angenommen werden. Geschieht dies nicht, stimmt die Vollversammlung über die Annahme ab.
+
+(3) Änderungsanträge können vor der Vollversammlung schriftlich bekanntgegeben und begründet werden.
+
+## § 7 Anträge zur Geschäftsordnung
+
+(1) Anträge zur Geschäftsordnung werden durch das Heben beider Arme angezeigt. Davon kann abgewichen werden, wenn die Intention für die Versammlungsleitung eindeutig erkennbar ist.
+
+(2) Anträge zur Geschäftsordnung werden unverzüglich behandelt.
+
+(3) Vor der Abstimmung darf je eine antragsberechtigte Person für bzw. gegen den Antrag das Wort erhalten.
+
+(4) Geschäftsordnungsanträge bedürfen einer Zweidrittelmehrheit der anwesenden Mitglieder der Vollversammlung.
+
+(5) Geschäftsordnungsanträge sind:
+  1. Antrag auf Vertagung eines Tagesordnungspunktes (einfache Mehrheit),
+  2. Antrag auf den Schluss der Redeliste,
+  3. Antrag auf den Schluss der Beratung und ggf. sofortige Abstimmung,
+  4. Antrag auf Ende der Sitzung,
+  5. Antrag auf Zählung und/oder Protokollierung des Abstimmungsergebnisses (auf Verlangen einer Person).
```

Comment:
ÄA: Ergänzung

6. Antrag auf Einführung einer Redezeitbegrenzung
7. Antrag auf Durchführung einer gehemein Abstimmung (10% der Sitzung)

Reply:
Zustimmung, habe dein 6. in meinem Vorschlag oben als mögliche Formen des Antrags nach Nr. 2 aufgefasst: https://github.com/fsr-de/docs/pull/30#discussion_r607371486

Re geheime Abstimmung: weiß nicht, ob 10% sinnvoll ist als Quorum oder nicht, das müsste mal wer beantworten, der weiß, wie viele Leute an so einer VV wirklich aktiv beteiligt sind. Wenn man Pech hat sind sehr viele da, aber nur wenige aktiv -> 10 % nie erreicht. Daher hatte ich ein Minderheitsquorum von 10 Leuten vorgeschlagen, aber alles diskutabel. Sollte aber insgesamt konsistent sein (also dass nur existieren: Minderheitsquorum X, einfache Mehrheit, 2/3-Mehrheit)

Reply:
Ich werde beide Punkte mit in die FSR-Runde nehmen
@luap42 Der GO-Antrag wird von einer Person gestellt. Alle Anwesenden stimmen dann darüber ab. Wortmeldungen kommen natürlich meist nur von wenigen Personen, aber eine VV ist auch der falsche Ort, um erstmals lange über alles zu diskutieren, das sollte vorher geschehen sein.
Ich bin gerade unsicher, was "aktiv Beteiligte" sind und weshalb ihre Anzahl relevant ist. Könntest du das bitte nochmal erläutern?

Reply:
@ADimeo Wir haben uns auf der letzten FSR-Sitzung dazu entschieden, eine sowohl den GO-Antrag auf geheime Abstimmung als auch einen für die Festsetzung der Redezeitbegrenzung einzuführen, siehe letzter Commit.
Kann dieser Thread damit geschlossen werdenß